build.sh

```diff
@@ -106,7 +106,7 @@ RunGulp()
     echo "##teamcity[progressFinish 'npm install']"
 
     echo "##teamcity[progressStart 'Running Gulp']"
-    CheckExitCode gulp build
+    CheckExitCode npm run gulp build
```

Comment:
I really like this, since it means we won't need to install gulp on the build servers.